packages/solara-vuetify-app/package.json

```
@@ -19,14 +19,14 @@
     "material-design-icons-iconfont": "^5.0.1",
     "typeface-roboto": "0.0.54",
     "vue": "~2.6.14",
-    "vuetify": "~2.2.26"
+    "vuetify": "~2.2.34"
```

```suggestion
    "vuetify": "~2.2.26"
```
We need to keep this in sync with https://github.com/widgetti/ipyvuetify/blob/778deaa9c289f9466f421b11f2042b73b251ffc3/js/package.json#L59

---

We had a discussion about this with @mariobuikhuizen back in the day, and if I remember correctly, thought that there isn't much harm that would come from bumping it up to another compatible patch version. Though maybe it is clearer to keep the versions matching, also as a reminder to bump up the version for ipyvuetify.